source/frontend/src/features/mapSideBar/acquisition/tabs/expropriation/ExpropriationTabContainerView.tsx

```diff
@@ -1,25 +1,39 @@
+import { useHistory, useRouteMatch } from 'react-router-dom';
 import { toast } from 'react-toastify';
 
 import LoadingBackdrop from '@/components/common/LoadingBackdrop';
 import { Section } from '@/components/common/Section/Section';
+import { SectionListHeader } from '@/components/common/SectionListHeader';
+import { Claims } from '@/constants';
+import { useForm8Repository } from '@/hooks/repositories/useForm8Repository';
 import { Api_AcquisitionFile, EnumAcquisitionFileType } from '@/models/api/AcquisitionFile';
+import { Api_ExpropriationPayment } from '@/models/api/ExpropriationPayment';
 
 import { useGenerateExpropriationForm1 } from '../../common/GenerateForm/hooks/useGenerateExpropriationForm1';
 import { useGenerateExpropriationForm5 } from '../../common/GenerateForm/hooks/useGenerateExpropriationForm5';
 import { useGenerateExpropriationForm9 } from '../../common/GenerateForm/hooks/useGenerateExpropriationForm9';
 import ExpropriationForm1 from './form1/ExpropriationForm1';
 import ExpropriationForm5 from './form5/ExpropriationForm5';
+import ExpropriationForm8Details from './form8/details/ExpropriationForm8Details';
 import ExpropriationForm9 from './form9/ExpropriationForm9';
 
 export interface IExpropriationTabContainerViewProps {
   loading: boolean;
   acquisitionFile: Api_AcquisitionFile;
+  form8s: Api_ExpropriationPayment[];
+  onForm8Deleted: (form8Id: number) => void;
 }
 
 export const ExpropriationTabContainerView: React.FunctionComponent<
   IExpropriationTabContainerViewProps
-> = ({ loading, acquisitionFile }) => {
-  // TODO: Load Form 8 into this container
+> = ({ loading, acquisitionFile, form8s, onForm8Deleted }) => {
+  const history = useHistory();
+  const match = useRouteMatch();
+
+  const {
```

Comment:
this isn't really the right place for this, the view shouldn't be making api requests, that should all get delegated to the container.

Reply:
views should essentially only ever be display logic, any kind of interactions should be delegated to the container. That way the view can be tested simply with the props and all of the mocking happens at the container level.

Reply:
updated